sympy/integrals/tests/test_integrals.py

```diff
@@ -2097,3 +2097,16 @@ def test_issue_20782():
     f = Piecewise((0, x < 0.0), (1, True)) - Piecewise((0, x < 1.0), (1, True))
     assert integrate(f, (x, -oo, 1)) == 1
     assert integrate(-f, (x, -oo, 1)) == -1
+
+
+def test_issue_19427():
+    # <https://github.com/sympy/sympy/issues/19427>
+    x = Symbol("x")
+
+    # Have always been okay:
+    # assert integrate((x ** 4) * sqrt(1 - x ** 2), (x, -1, 1)) == pi / 16
```

Comment:
It should not be an issue if you add the correct tests also as regression tests, showing new modifications haven't negatively impacted working examples. You can uncomment these lines.

Reply:
Seems a little weird to me. If "the more the better" is a test strategy, why not also test `assert integrate((x ** 2) * sqrt(1 - x ** 2), (x, -1, 1)) == pi / 8`?